experimental/plugins_graphdriver.md

```diff
@@ -278,9 +278,9 @@ Responds with a list of changes. The structure of a change is:
 Where the `Path` is the filesystem path within the layered filesystem that is
 changed and `Kind` is an integer specifying the type of change that occurred:
 
-- 0 - Modified
-- 1 - Added
-- 2 - Deleted
+- 1 - Modified
+- 2 - Added
+- 3 - Deleted
```

Comment:
have you verified that these new values actually match the source code? This is not just a numbered list